src/components/header.js

```diff
@@ -49,7 +53,11 @@ function Header() {
           </svg>
         </button>
 
-        <nav className={`${isExpanded ? `block` : `hidden`} navMenu`}>
+        <nav
+          className={
+            `${isExpanded ? `block` : `hidden`} ` + componentStyles.navMenu
```

Comment:
I'm going to revisit how ternary operators are used to better understand what's going on on this line.